### PR TITLE
Merge the MC Version, Modloader Type, and Modloader Version Boxes

### PR DESCRIPTION
**Pull request by furtabs**
## Purpose
Cleans up the GDLauncher UI By Unifying Game Versions and Modloader Strings.

## Approach
Before:
image

After: 
image
image
image

#### Open Questions and Pre-Merge TODOs
not applicable

## Learning
not applicable

#### Blog Posts
sorry im confused

### CODE REVIEW

File: src/common/modals/InstanceManager/Overview.js [hunk 1]
```diff
@@ -261,42 +261,30 @@ const Overview = ({ instanceName, background, manifest }) => {
...
-                <FontAwesomeIcon icon={faCog} />
-              ) : null
-            }
-          >
-            {config?.modloader[0] === 'forge'
-              ? config?.modloader[2]?.split('-')[1]
-              : config?.modloader[2] || '-'}
+            {config?.modloader[0] === "fabric" ? " - " : ""}
+            {config?.modloader[0] === "fabric"
+              ? config?.modloader[2] || "-"
```
Comment (killpowa) [inline src/common/modals/InstanceManager/Overview.js:282]: Codacy found an issue: [Replace `"-"` with `'-'`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

**Comment by killpowa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 16
           

Complexity increasing per file
==============================
- src/common/modals/InstanceManager/Overview.js  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493&bid=20414975)

File: src/common/modals/InstanceManager/Overview.js [hunk 2]
```diff
@@ -308,27 +296,27 @@ const Overview = ({ instanceName, background, manifest }) => {
...
           >
             {convertMinutesToHumanTime(config?.timePlayed)}
           </Card>
           <Card
             title="Last Played"
-            color={props => props.theme.palette.colors.orange}
+            color={(props) => props.theme.palette.colors.orange}
           >
-            {config?.lastPlayed ? computeLastPlayed(config?.lastPlayed) : '-'}
+            {config?.lastPlayed ? computeLastPlayed(config?.lastPlayed) : "-"}
```
Comment (killpowa) [inline src/common/modals/InstanceManager/Overview.js:314]: Codacy found an issue: [Replace `"-"` with `'-'`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js [hunk 3]
```diff
@@ -261,42 +261,30 @@ const Overview = ({ instanceName, background, manifest }) => {
...
-            width: 100%;
+            width: 100% + 20px;
             margin-top: 20px;
+            margin-left: -20px;
           `}
         >
           <Card
             title="Minecraft Version"
-            color={props => props.theme.palette.colors.jungleGreen}
+            color={(props) => props.theme.palette.colors.darkYellow}
```
Comment (killpowa) [inline src/common/modals/InstanceManager/Overview.js:271]: Codacy found an issue: [Replace `(props)` with `props`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js [hunk 4]
```diff
@@ -261,42 +261,30 @@ const Overview = ({ instanceName, background, manifest }) => {
...
-            icon={
-              (config?.modloader[2] || '-') !== '-' ? (
-                <FontAwesomeIcon icon={faCog} />
-              ) : null
-            }
-          >
-            {config?.modloader[0] === 'forge'
-              ? config?.modloader[2]?.split('-')[1]
-              : config?.modloader[2] || '-'}
+            {config?.modloader[0] === "fabric" ? " - " : ""}
```
Comment (killpowa) [inline src/common/modals/InstanceManager/Overview.js:280]: Codacy found an issue: [Replace `"fabric"·?·"·-·"·:·""` with `'fabric'·?·'·-·'·:·''`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js [hunk 5]
```diff
@@ -308,27 +296,27 @@ const Overview = ({ instanceName, background, manifest }) => {
           `}
         >
           <Card
-            title="Mods"
-            color={props => props.theme.palette.colors.maximumRed}
+            title="Mod Count"
+            color={(props) => props.theme.palette.colors.maximumRed}
```
Comment (killpowa) [inline src/common/modals/InstanceManager/Overview.js:300]: Codacy found an issue: [Replace `(props)` with `props`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js [hunk 6]
```diff
@@ -261,42 +261,30 @@ const Overview = ({ instanceName, background, manifest }) => {
...
-          >
-            {config?.modloader[0] === 'forge'
-              ? config?.modloader[2]?.split('-')[1]
-              : config?.modloader[2] || '-'}
+            {config?.modloader[0] === "fabric" ? " - " : ""}
+            {config?.modloader[0] === "fabric"
+              ? config?.modloader[2] || "-"
+              : ""}
+            {config?.modloader[0] === "forge" ? " - " : ""}
+            {config?.modloader[0] === "forge"
```
Comment (killpowa) [inline src/common/modals/InstanceManager/Overview.js:285]: Codacy found an issue: [Replace `"forge"` with `'forge'`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js [hunk 7]
```diff
@@ -308,27 +296,27 @@ const Overview = ({ instanceName, background, manifest }) => {
...
+            title="Mod Count"
+            color={(props) => props.theme.palette.colors.maximumRed}
           >
-            {config?.mods?.length || '-'}
+            {config?.mods?.length || "-"}
           </Card>
           <Card
             title="Played Time"
-            color={props => props.theme.palette.colors.liberty}
+            color={(props) => props.theme.palette.colors.liberty}
```
Comment (killpowa) [inline src/common/modals/InstanceManager/Overview.js:306]: Codacy found an issue: [Replace `(props)` with `props`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js [hunk 8]
```diff
@@ -261,42 +261,30 @@ const Overview = ({ instanceName, background, manifest }) => {
...
         >
           <Card
             title="Minecraft Version"
-            color={props => props.theme.palette.colors.jungleGreen}
+            color={(props) => props.theme.palette.colors.darkYellow}
             instanceName={instanceName}
             defaultValue={config?.modloader}
             icon={<FontAwesomeIcon icon={faCog} />}
           >
+            {config?.modloader[0] === "vanilla" ? "Vanilla " : ""}
```
Comment (killpowa) [inline src/common/modals/InstanceManager/Overview.js:276]: Codacy found an issue: [Replace `"vanilla"·?·"Vanilla·"·:·""` with `'vanilla'·?·'Vanilla·'·:·''`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js [hunk 9]
```diff
@@ -308,27 +296,27 @@ const Overview = ({ instanceName, background, manifest }) => {
           `}
         >
           <Card
-            title="Mods"
-            color={props => props.theme.palette.colors.maximumRed}
+            title="Mod Count"
+            color={(props) => props.theme.palette.colors.maximumRed}
           >
-            {config?.mods?.length || '-'}
+            {config?.mods?.length || "-"}
```
Comment (killpowa) [inline src/common/modals/InstanceManager/Overview.js:302]: Codacy found an issue: [Replace `"-"` with `'-'`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js [hunk 10]
```diff
@@ -261,42 +261,30 @@ const Overview = ({ instanceName, background, manifest }) => {
...
-              (config?.modloader[2] || '-') !== '-' ? (
-                <FontAwesomeIcon icon={faCog} />
-              ) : null
-            }
-          >
-            {config?.modloader[0] === 'forge'
-              ? config?.modloader[2]?.split('-')[1]
-              : config?.modloader[2] || '-'}
+            {config?.modloader[0] === "fabric" ? " - " : ""}
+            {config?.modloader[0] === "fabric"
```
Comment (killpowa) [inline src/common/modals/InstanceManager/Overview.js:281]: Codacy found an issue: [Replace `"fabric"` with `'fabric'`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js [hunk 11]
```diff
@@ -261,42 +261,30 @@ const Overview = ({ instanceName, background, manifest }) => {
...
             title="Minecraft Version"
-            color={props => props.theme.palette.colors.jungleGreen}
+            color={(props) => props.theme.palette.colors.darkYellow}
             instanceName={instanceName}
             defaultValue={config?.modloader}
             icon={<FontAwesomeIcon icon={faCog} />}
           >
+            {config?.modloader[0] === "vanilla" ? "Vanilla " : ""}
+            {config?.modloader[0] === "fabric" ? "Fabric " : ""}
+            {config?.modloader[0] === "forge" ? "Forge " : ""}
```
Comment (killpowa) [inline src/common/modals/InstanceManager/Overview.js:278]: Codacy found an issue: [Replace `"forge"·?·"Forge·"·:·""` with `'forge'·?·'Forge·'·:·''`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js [hunk 12]
```diff
@@ -308,27 +296,27 @@ const Overview = ({ instanceName, background, manifest }) => {
...
             title="Played Time"
-            color={props => props.theme.palette.colors.liberty}
+            color={(props) => props.theme.palette.colors.liberty}
           >
             {convertMinutesToHumanTime(config?.timePlayed)}
           </Card>
           <Card
             title="Last Played"
-            color={props => props.theme.palette.colors.orange}
+            color={(props) => props.theme.palette.colors.orange}
```
Comment (killpowa) [inline src/common/modals/InstanceManager/Overview.js:312]: Codacy found an issue: [Replace `(props)` with `props`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js [hunk 13]
```diff
@@ -261,42 +261,30 @@ const Overview = ({ instanceName, background, manifest }) => {
...
-              ) : null
-            }
-          >
-            {config?.modloader[0] === 'forge'
-              ? config?.modloader[2]?.split('-')[1]
-              : config?.modloader[2] || '-'}
+            {config?.modloader[0] === "fabric" ? " - " : ""}
+            {config?.modloader[0] === "fabric"
+              ? config?.modloader[2] || "-"
+              : ""}
```
Comment (killpowa) [inline src/common/modals/InstanceManager/Overview.js:283]: Codacy found an issue: [Replace `""` with `''`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js [hunk 14]
```diff
@@ -261,42 +261,30 @@ const Overview = ({ instanceName, background, manifest }) => {
...
+            {config?.modloader[0] === "forge" ? " - " : ""}
+            {config?.modloader[0] === "forge"
+              ? config?.modloader[2]?.split("-")[1]
```
Comment (killpowa) [inline src/common/modals/InstanceManager/Overview.js:286]: Codacy found an issue: [Replace `"-"` with `'-'`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js [hunk 15]
```diff
@@ -261,42 +261,30 @@ const Overview = ({ instanceName, background, manifest }) => {
...
           <Card
             title="Minecraft Version"
-            color={props => props.theme.palette.colors.jungleGreen}
+            color={(props) => props.theme.palette.colors.darkYellow}
             instanceName={instanceName}
             defaultValue={config?.modloader}
             icon={<FontAwesomeIcon icon={faCog} />}
           >
+            {config?.modloader[0] === "vanilla" ? "Vanilla " : ""}
+            {config?.modloader[0] === "fabric" ? "Fabric " : ""}
```
Comment (killpowa) [inline src/common/modals/InstanceManager/Overview.js:277]: Codacy found an issue: [Replace `"fabric"·?·"Fabric·"·:·""` with `'fabric'·?·'Fabric·'·:·''`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js [hunk 16]
```diff
@@ -261,42 +261,30 @@ const Overview = ({ instanceName, background, manifest }) => {
...
-            }
-          >
-            {config?.modloader[0] === 'forge'
-              ? config?.modloader[2]?.split('-')[1]
-              : config?.modloader[2] || '-'}
+            {config?.modloader[0] === "fabric" ? " - " : ""}
+            {config?.modloader[0] === "fabric"
+              ? config?.modloader[2] || "-"
+              : ""}
+            {config?.modloader[0] === "forge" ? " - " : ""}
```
Comment (killpowa) [inline src/common/modals/InstanceManager/Overview.js:284]: Codacy found an issue: [Replace `"forge"·?·"·-·"·:·""` with `'forge'·?·'·-·'·:·''`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

**Comment by furtabs** [comment]
Come on i just did what ESLint told me to do

**Comment by blarfoon** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 16
           

Complexity increasing per file
==============================
- src/common/modals/InstanceManager/Overview.js  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493&bid=20414975)

File: src/common/modals/InstanceManager/Overview.js (hunk 5 above)
Comment (blarfoon) [inline src/common/modals/InstanceManager/Overview.js:300]: Codacy found an issue: [Replace `(props)` with `props`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js (hunk 14 above)
Comment (blarfoon) [inline src/common/modals/InstanceManager/Overview.js:286]: Codacy found an issue: [Replace `"-"` with `'-'`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js (hunk 8 above)
Comment (blarfoon) [inline src/common/modals/InstanceManager/Overview.js:276]: Codacy found an issue: [Replace `"vanilla"·?·"Vanilla·"·:·""` with `'vanilla'·?·'Vanilla·'·:·''`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js (hunk 11 above)
Comment (blarfoon) [inline src/common/modals/InstanceManager/Overview.js:278]: Codacy found an issue: [Replace `"forge"·?·"Forge·"·:·""` with `'forge'·?·'Forge·'·:·''`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js (hunk 13 above)
Comment (blarfoon) [inline src/common/modals/InstanceManager/Overview.js:283]: Codacy found an issue: [Replace `""` with `''`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js (hunk 10 above)
Comment (blarfoon) [inline src/common/modals/InstanceManager/Overview.js:281]: Codacy found an issue: [Replace `"fabric"` with `'fabric'`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js (hunk 2 above)
Comment (blarfoon) [inline src/common/modals/InstanceManager/Overview.js:314]: Codacy found an issue: [Replace `"-"` with `'-'`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js (hunk 3 above)
Comment (blarfoon) [inline src/common/modals/InstanceManager/Overview.js:271]: Codacy found an issue: [Replace `(props)` with `props`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js (hunk 9 above)
Comment (blarfoon) [inline src/common/modals/InstanceManager/Overview.js:302]: Codacy found an issue: [Replace `"-"` with `'-'`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js (hunk 1 above)
Comment (blarfoon) [inline src/common/modals/InstanceManager/Overview.js:282]: Codacy found an issue: [Replace `"-"` with `'-'`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js (hunk 4 above)
Comment (blarfoon) [inline src/common/modals/InstanceManager/Overview.js:280]: Codacy found an issue: [Replace `"fabric"·?·"·-·"·:·""` with `'fabric'·?·'·-·'·:·''`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js (hunk 15 above)
Comment (blarfoon) [inline src/common/modals/InstanceManager/Overview.js:277]: Codacy found an issue: [Replace `"fabric"·?·"Fabric·"·:·""` with `'fabric'·?·'Fabric·'·:·''`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js (hunk 6 above)
Comment (blarfoon) [inline src/common/modals/InstanceManager/Overview.js:285]: Codacy found an issue: [Replace `"forge"` with `'forge'`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js (hunk 7 above)
Comment (blarfoon) [inline src/common/modals/InstanceManager/Overview.js:306]: Codacy found an issue: [Replace `(props)` with `props`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js (hunk 16 above)
Comment (blarfoon) [inline src/common/modals/InstanceManager/Overview.js:284]: Codacy found an issue: [Replace `"forge"·?·"·-·"·:·""` with `'forge'·?·'·-·'·:·''`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js (hunk 12 above)
Comment (blarfoon) [inline src/common/modals/InstanceManager/Overview.js:312]: Codacy found an issue: [Replace `(props)` with `props`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6271493)

File: src/common/modals/InstanceManager/Overview.js [hunk 17]
```diff
@@ -261,42 +261,30 @@ const Overview = ({ instanceName, background, manifest }) => {
           css={`
             display: flex;
             justify-content: space-between;
-            width: 100%;
+            width: 100% + 20px;
```
Comment (Ladvace) [inline src/common/modals/InstanceManager/Overview.js:264]: does it works? Probably it's better using calc(100%+20px);

**Comment by Ladvace** [comment]
actually I don't like it, I don't know if it's for the color  but I think that having a giant horizontal rectangle doesn't make it cleaner

**Comment by furtabs** [comment]
The curse Forge box is the same way

**Comment by furtabs** [comment]
And it is definally cleaner

**Comment by blarfoon** [comment]
This PR is not passing any check so it cannot be merged for that in the first place.
Moreover, I don't like the look of having one giant bar at the top with only that info, and it makes it less obvious what you're looking at but let's see what other people say about this

**Pull request closed without merging**